lib/models/asset-size-printer.js

```diff
@@ -39,16 +38,14 @@ AssetPrinterSize.prototype.makeAssetSizesObject = function () {
   var assets = files
     // Skip test files
     .filter(function (file) {
-      var filename = path.basename(file);
-      return !testFileRegex.test(filename);
+      return !testFileRegex.test(file);
```

Comment:
I don't believe that this section needs to change? We really want that to compare at the file name level so that we don't get false positives on path matches.